frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/result-tab/error-results.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Title,
+} from '@patternfly/react-core';
+import { iGetCommonData, iGetCreateVMWizardTabs } from '../../selectors/immutable/selectors';
+import { VMWizardProps, VMWizardTab } from '../../types';
+import { makeSentence } from '../../../../utils/grammar';
+import { getVMLikeModelName } from '../../../../utils/utils';
+import { iGetIn } from '../../../../utils/immutable';
+
+const ErrorResultsComponent: React.FC<ErrorResultsProps> = ({
+  isCreateTemplate,
+  mainError,
+  className,
+}) => {
+  const modelName = getVMLikeModelName(isCreateTemplate);
+
+  return (
+    <EmptyState variant={EmptyStateVariant.full} className={className}>
+      <EmptyStateIcon icon={ExclamationCircleIcon} color="#a30000" />
+      <Title headingLevel="h5" size="lg">
+        {makeSentence(`Error creating ${modelName}`)}
```

Comment:
I don't see benefit in using `makeSentence` function for hardcoded strings. I believe we can make sentence from them directly.

Reply:
You are right - this was a copy paste from `pending-results`